usaspending_api/etl/subaward_etl.py

```diff
@@ -18,6 +18,7 @@
                                          AND fsrs_procurement.idv_reference_number = award_procurement.parent_award_id
        LEFT OUTER JOIN fsrs_subcontract  ON fsrs_subcontract.parent_id = fsrs_procurement.id
 WHERE  award_procurement.submission_id = %s
+       AND award_procurement.piid IN %s
```

Comment:
I haven't been following along--do we need to care about parent award id for the purposes of this lookup?